app/helpers/tagging_helper.rb

```diff
@@ -1,10 +1,11 @@
 module TaggingHelper
   def taggable_vita_partners(vita_partners)
     taggable_vita_partners = []
-    vita_partners.organizations.each do |vita_partner|
-      taggable_vita_partners << { id: vita_partner.id, name: vita_partner.name, value: vita_partner.id }
-      vita_partner.child_sites.each do |site|
-        taggable_vita_partners << { id: site.id, name: site.name, parentName: vita_partner.name, value: site.id }
+    # We query via the Organization model to trigger auto-loading of child_sites per Organization model's default scope.
```

Comment:
Good idea. I'll keep that in mind if we see other places that might need this treatment. One oddity is that `@vita_partners` is an access-controlled collection of sites _and_ organizations; if it were `@organizations` I'd definitely agree it would make sense to do the preload there.